Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -69,9 +74,12 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
    */
   UPROPERTY(
       EditAnywhere,
+      BlueprintReadOnly,
```

Comment:
Rather than making this read-only and adding a separate Set function, could we use the `BlueprintSetter` specifier?
https://docs.unrealengine.com/en-US/ProgrammingAndScripting/GameplayArchitecture/Properties/Specifiers/index.html
(I've never tried, so the answer could very well be no)